src/charm_state.py

```diff
@@ -514,6 +521,8 @@ def from_charm(cls, charm: CharmBase) -> "CharmConfig":
             token=github_config.token if github_config else None,
             manager_proxy_command=manager_proxy_command,
             use_aproxy=use_aproxy,
+            aproxy_exclude_addresses=charm.config.get(APROXY_EXCLUDE_ADDRESSES_CONFIG_NAME),
+            aproxy_redirect_ports=charm.config.get(APROXY_REDIRECT_PORTS_CONFIG_NAME),
```

Comment:
we should validate the format somewhere to block directly , e.g. if an operator specifies an invalid format

Reply:
Yes, added some validation to these two fields, thanks!